README.md

```diff
@@ -83,6 +83,7 @@ This library uses C++17 (or newer when available).
 | CUDA (clang)                                                                       | -                                         | -                                         | -                                         | -                                    | -                      | -                       | -                                         | :x:                                       | :white_check_mark: <br/> (CUDA 9.2-10.1)  | :white_check_mark: <br/> (CUDA 10.0-10.2) | -                                    | -                                                     | -                                         | -                                    |
 | [HIP-4.2](https://alpaka.readthedocs.io/en/latest/install/HIP.html) (clang)        | :x:                                       | :x:                                       | :x:                                       | :x:                                  | :x:                    | :x:                     | :x:                                       | :x:                                       | :x:                                       | :white_check_mark:                        | -                                    | -                                                     | -                                         | -                                    |
 | [HIP-4.3 and 4.5](https://alpaka.readthedocs.io/en/latest/install/HIP.html) (clang)| :x:                                       | :x:                                       | :x:                                       | :x:                                  | :x:                    | :x:                     | :x:                                       | :x:                                       | :x:                                       | :x:                                       | :white_check_mark:                   | -                                                     | -                                         | -                                    |
+| [HIP-5.0](https://alpaka.readthedocs.io/en/latest/install/HIP.html) (clang)| :x:                                       | :x:                                       | :x:                                       | :x:                                  | :x:                    | :x:                     | :x:                                       | :x:                                       | :x:                                       | :x:                                       | :white_check_mark:                   | -                                                     | -                                         | -                                    |
```

Comment:
I think we should merge the table rows similar to how we do it with CUDA.

Reply:
If the CI passes I'll do that in a follow-up PR. We should have a talk about supported HIP versions anyway...